server/game/cards/01-DTR/BBAttorneys.js

```diff
@@ -0,0 +1,62 @@
+const DeedCard = require('../../deedcard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class BBAttorneys extends DeedCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'B & B Attorneys',
+            playType: ['noon'],
+            cost: ability.costs.bootSelf(),
+            target: {
+                activePromptTitle: 'Select a wanted dude to raise or lower their bounty by one.',
```

Comment:
```suggestion
                activePromptTitle: 'Select a wanted dude',
```
I would make it shorter, you should know what you are doing based on the cards you are using.

Reply:
handled. :heavy_check_mark: